### PR TITLE
[Keyboard] IDOBAO ID80v1 folder rename

### PR DESCRIPTION
**Pull request by vinorodrigues**
## Description

IDOBAO support has received numerous support calls with users confused with the "v1" name used in the folder structure introduced in https://github.com/qmk/qmk_firmware/pull/15992.

The code in the `v1` folder is a folder shift from the base of the `id80`folder and actually pertains to the "v2" boards. Both "v1" *(with no per-key under-light, or rather; self-solder per-key)* and "v2" *(with manufactured white per-key leds)* are compatible with that code base - however users seeking code for their "v2" boards are using the "v3" codebase, and this is not compatible as it uses RGB LED's and the RGB Matrix codebase.

This PR seeks to rename the "v1" folder to "v2" *(to eliminate confusion)*, and then add a `readme.md` and `rules.mk` file to the "v1" (ensuring backward compatibility).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A *(more about code clean-up)*

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/idobao/id80/v1/readme.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
 # IDOBAO ID80
 
-![ID80](https://ae01.alicdn.com/kf/H0f617c6129c24cfaa26a8c9e8e54851aj.jpg)
+![ID80](https://i.imgur.com/977ENjp.png)
```
Comment (drashna) [inline keyboards/idobao/id80/v1/readme.md:3]:
```suggestion
![ID80](https://i.imgur.com/977ENjph.png)
```

File: keyboards/idobao/id80/v2/readme.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# IDOBAO ID80
+
+![ID80](https://i.imgur.com/977ENjp.png)
```
Comment (drashna) [inline keyboards/idobao/id80/v2/readme.md:3]:
```suggestion
![ID80](https://i.imgur.com/977ENjph.png)
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/idobao/id80/v2/rules.mk [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes        # Mouse keys
+EXTRAKEY_ENABLE = yes        # Audio control and System control
+CONSOLE_ENABLE = no          # Console for debug
+COMMAND_ENABLE = no          # Commands for debug and configuration
+NKRO_ENABLE = yes            # Enable N-Key Rollover
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+BACKLIGHT_ENABLE = yes       # Enable underkey LED backlight
```
Comment (fauxpark) [inline keyboards/idobao/id80/v2/rules.mk:20]:
```suggestion
# MCU name
MCU = atmega32u4

# Bootloader selection
BOOTLOADER = atmel-dfu

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes       # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes        # Mouse keys
EXTRAKEY_ENABLE = yes        # Audio control and System control
CONSOLE_ENABLE = no          # Console for debug
COMMAND_ENABLE = no          # Commands for debug and configuration
NKRO_ENABLE = yes            # Enable N-Key Rollover
RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
BACKLIGHT_ENABLE = yes       # Enable underkey LED backlight

DEFAULT_FOLDER = idobao/id80/v2/ansi
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
Have these been checked to make sure they still compile?

**Comment by vinorodrigues** [comment]
absolument

image

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**